platform/ios/export/export_plugin.cpp

```diff
@@ -1954,4 +1954,6 @@ EditorExportPlatformIOS::~EditorExportPlatformIOS() {
 	quit_request.set();
 	check_for_changes_thread.wait_to_finish();
 #endif
+
+	env_script_encryption_key = "GODOT_IOS_SCRIPT_ENCRYPTION_KEY";
```

Comment:
Similar to the comment in the Android section; I'm guessing this should be in the constructor, not the destructor?

Reply:
Same as above. I'll take care of it when I rebase to resolve the current batch of merge conflicts.